sample-operators/mysql-schema/src/main/java/io/javaoperatorsdk/operator/sample/dependent/SchemaDependentResource.java

```diff
@@ -30,9 +33,11 @@
 @SchemaConfig(pollPeriod = 700, host = "127.0.0.1",
     port = SchemaDependentResource.LOCAL_PORT,
     user = "root", password = "password") // NOSONAR: password is only used locally, example only
+@Configured(by = SchemaConfig.class, with = ResourcePollerConfig.class,
```

Comment:
Note sure I'm following? Are you saying that the `Configured` annotation should annotate the `SchemaConfig` annotation? If yes, I'm not sure how it would work… Maybe something to explore in a second step? That said, considering that this feature is probably not going to be used that much and it's already quite tricky to support I'm a little reluctant to make it even more complex to support.